src/components/common/Flex/style.css.ts

```diff
@@ -0,0 +1,11 @@
+import { style } from '@vanilla-extract/css';
+
+export const flex = style({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  rowGap: '10px',
+  columnGap: '10px',
+  margin: '0 80px',
+  padding: '10px',
+});
```

Comment:
```suggestion
import * as utils from '@/src/styles/utils.css.ts';

export const flex = style(
[
  utils.flexCenter
], 
{
  rowGap: '10px',
  columnGap: '10px',
  margin: '0 80px',
  padding: '10px',
});
```

이렇게 미리 정의된 유틸을 사용하면 단순하게 반복되는 스타일 속성들을 줄일 수 있어요!